### PR TITLE
Updated README

### LINKED ISSUE #16: Incomplete README.
@jatinkatyal13 There must be proper README for this project. there is no information given about this project at all.

### PR DESCRIPTION
**Pull request by prathambatra**
Signed-off-by: prathambatra <batrapratham999@gmail.com>
fixes #16

### CODE REVIEW

**Comment by prathambatra** [comment]
@jatinkatyal13 review please.

File: src/config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   host: process.env.host || "localhost",
-  login: process.env.login || "test",
-  password: process.env.password || "test",
+  login: process.env.login || "guest",
```
Comment (jatinkatyal13) [inline src/config.js:3]: We can probably ignore this as the default creds are test in docker-compose and in `.env.sample` file

Comment (jatinkatyal13) [reply src/config.js:3]: With ignore I mean revert the changes using
`git checkout origin/master -- src/config.hs`

**Comment by jatinkatyal13** [comment]
Nice Work! Just one little change

File: src/config.js (hunk 1 above)
Comment (prathambatra) [reply src/config.js:3]: okay

**Comment by prathambatra** [comment]
@jatinkatyal13 done

**Pull request merged**